### ISSUE TITLE
for test code

### ISSUE THREAD

**Issue by runzeer** [waxz/sfm_net#1]
Could you share your test code? I have tried to write the test code according to your training but I found that the generated depth is equal to 1 no matter how much epochs I have trained? So could you share your test code for learning?

**Comment by shahyaan** [comment]
I think this is incorrect and the reason why the depth value is always 1.
def clip_relu(x):

    x=tf.clip_by_value(x, 1, 100)
    
    return x
remove this and use a regular relu activation in the depth_net like this:

def depth_net(frame):

    top,_=conv_deconv_net(frame)
    top=Conv2D(filters=1,kernel_size=1,strides=1,padding='same',kernel_initializer=keras.initializers.glorot_normal())(top)
    depth=Activation(relu)(top)
    depth = tf.clip_by_value(depth, 0,100) # this limits predictions to 100 as stated in paper
    return depth

Also initialize the bias for the 1x1 convolution layer in the depth_net to be 1

**Comment by martin96depe** [comment]
Hi, sorry I am quite new in deep learning but can you tell how can I use the trained model and display the depth output? I have trained the model and I would like to see if the predicted depth output is good.

Thanks for helping.
